devel/gradle/Portfile

```diff
@@ -23,6 +23,18 @@ master_sites        https://services.gradle.org/distributions
 checksums           rmd160  91ddb4be28b859e39166526228d39bbaabf91100 \
                     sha256  b49c6da1b2cb67a0caf6c7480630b51c70a11ca2016ff2f555eaeda863143a29 \
                     size    78420037
+                    
+subport ${name}-devel {
```

Comment:
Please fix the indentation. Also you don't need to list anything here that is the same as the parent port (so no `homepage`, `platforms`, etc.).

Reply:
The livecheck regex needs to be different for gradle-devel. Right now it shares a definition with gradle, and it fails:

```
$ port livecheck gradle-devel
Error: livecheck failed for gradle-devel: extracted version '4.10.2' is older than livecheck.version '5.0-milestone-1
```

Reply:
Also, to be honest, I'm not interested in maintaining gradle-devel, so please add a `maintainers` line with either yourself (and `openmaintainer` if you like) or `nomaintainer`.